guides/pages/how-to-test-webhooks-on-localhost.mdx

```diff
@@ -0,0 +1,280 @@
+---
+meta:
+  title: "How to test webhooks on localhost"
+  description:
+    "Learn how to test Liveblocks webhooks on your local system using
+    localtunnel or ngrok."
+---
+
+Testing webhooks on your local system can be difficult, but there are ways to
+make it possible using tools such as
+[`localtunnel`](https://www.npmjs.com/package/localtunnel) and
+[`ngrok`](https://www.npmjs.com/package/ngrok).
+
+## Create an endpoint in your project
+
+The first step in testing webhooks is making sure you have an API endpoint set
+up in your project. This is the endpoint that’ll receive the webhook event from
+Liveblocks.
+
+In order to use webhooks, we’ll need to retrieve the `headers` and `body` from
+the request. Here’s the basic endpoint we’ll be starting from:
+
+```ts
+export async function POST(request: Request) {
+  const body = await request.json();
+  const headers = request.headers;
+
+  // Handle webhooks
+  // ...
+
+  return new Response(null, { status: 200 });
+}
+```
+
+Create this endpoint in your project, and make it available on `localhost` at
+the following URL:
+
+```
+/api/liveblocks-database-sync
+```
+
+## Testing webhooks locally
+
+Tools such as `localtunnel` and `ngrok` allow you to temporarily place your
+localhost server online, by providing you with a temporary URL. Let’s take a
+look at these two options.
+
+### localtunnel
+
+[`localtunnel`](https://www.npmjs.com/package/localtunnel) allows you to get
+started without signing up. If your project is running on `localhost:3000`, you
+can run the following `localtunnel` command to generate your URL, which will
+stay online while your localhost server is running:
+
+```bash
+npx localtunnel --port 3000
+```
+
+Visit the URL that localtunnel creates, and correctly input your IP address as
+it requests. The URL `localtunnel` generates can be placed into the Liveblocks
+webhooks dashboard for quick testing.
+
+### ngrok
+
+[`ngrok`](https://www.npmjs.com/package/ngrok) requires you to sign up and
+install, but it has more features and is simpler to use after you’ve created an
+account. If your project is running on `localhost:3000`, you can run the
+following `ngrok` command to generate your URL, which will stay online while
+your localhost server is running:
+
+```bash
+ngrok http 3000
+```
+
+The URL is generates can be placed into the Liveblocks webhooks dashboard for
+quick testing.
+
+## Set up webhooks on the Liveblocks dashboard
+
+To use webhooks, you need to pass your endpoint URL to the webhooks dashboard
+inside your Liveblocks project, and tell the webhook to trigger when Storage
+document data has changed.
```

Comment:
Since the guide is generic I think we should not mention a single particular use-case.

```suggestion
inside your Liveblocks project, and tell the webhook to trigger on any specific [webhook events](/docs/platform/webhooks#Liveblocks-events).
```